src/core/public/chrome/chrome_service.tsx

```diff
@@ -40,6 +40,7 @@ const IS_LOCKED_KEY = 'core.chrome.isLocked';
 
 interface ConstructorParams {
   browserSupportsCsp: boolean;
+  packageInfoVersion: string;
```

Comment:
NIT: I think we usually name this value simply `kibanaVersion`

Reply:
Thanks! [Naming is hard.]([url](https://github.com/elastic/kibana/pull/102443/files#diff-8bb0fa43e19d24795e11f84500c3a2a9cff3275a5f89fe3c47cf2fb156c493c6R114))